checker-qual/src/main/java/org/checkerframework/checker/calledmethodsonelements/qual/EnsuresCalledMethodsOnElementsOnException.java

```diff
@@ -0,0 +1,77 @@
+package org.checkerframework.checker.calledmethodsonelements.qual;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.InheritedAnnotation;
+
+/**
+ * Indicates that the method, if it terminates by throwing an {@link Exception}, always invokes the
```

Comment:
I think this Javadoc should explicitly state that this annotation doesn't say anything about the annotated method's non-exceptional behavior, and that it's common to combine this annotation with an `@EnsuresCalledMethodsOnElements` annotation with the same arguments when specifying a method that _always_ calls another method (e.g., in a `finally` block before throwing).